<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="pbxcore"
           tests="50"
           failures="4"
           errors="0"
           skipped="0"
           time="1897.749">
  <properties />
            <testcase name="C4_checkTrunk2" time="102.927" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber">
                <failure type="java.lang.AssertionError"
                  message="CDR呼叫方检测 【预期：1100 [1100] 实际：1100dial1234567890123456789012345678901 [Emergency]】"
              >
<![CDATA[java.lang.AssertionError: CDR呼叫方检测 【预期：1100 [1100] 实际：1100dial1234567890123456789012345678901 [Emergency]】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:262)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber.C4_checkTrunk2(EmergencyNumber.java:285)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="E6_checkPhyAndPhy" time="74.244" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber">
                <failure type="java.lang.AssertionError"
                  message="CDR目的中继检测 【预期：BRI1 实际：FXO7】"
              >
<![CDATA[java.lang.AssertionError: CDR目的中继检测 【预期：BRI1 实际：FXO7】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:269)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber.E6_checkPhyAndPhy(EmergencyNumber.java:721)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="F2_EventCenter5" time="54.192" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber">
                <failure type="com.codeborne.selenide.ex.ElementShould"
                  message="Element should be visible {By.xpath: .//span[starts-with(@class,"x-btn-inner x-btn-inner-default-small") and text()="Search"]}
Element: '<span class="x-btn-inner x-btn-inner-default-small" data-ref="btnInnerEl" id="st-el-search-btnInnerEl" unselectable="on" displayed:false></span>'"
              >
<![CDATA[Element should be visible {By.xpath: .//span[starts-with(@class,"x-btn-inner x-btn-inner-default-small") and text()="Search"]}
Element: '<span class="x-btn-inner x-btn-inner-default-small" data-ref="btnInnerEl" id="st-el-search-btnInnerEl" unselectable="on" displayed:false></span>'
Screenshot: file:/F:/MyProject/SwebTest/build/reports/tests/1524350487760.5.png
Timeout: 15 s.
  com.codeborne.selenide.impl.WebElementSource.checkCondition(WebElementSource.java:66)
  com.codeborne.selenide.impl.WebElementSource.findAndAssertElementIsVisible(WebElementSource.java:72)
  com.codeborne.selenide.commands.Click.execute(Click.java:14)
  com.codeborne.selenide.commands.Click.execute(Click.java:11)
  com.codeborne.selenide.commands.Commands.execute(Commands.java:140)
  com.codeborne.selenide.impl.SelenideElementProxy.dispatchAndRetry(SelenideElementProxy.java:86)
  com.codeborne.selenide.impl.SelenideElementProxy.invoke(SelenideElementProxy.java:62)
  com.sun.proxy.$Proxy6.click(Unknown Source)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:260)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber.F2_EventCenter5(EmergencyNumber.java:900)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="E4_checkPhyAndVoip" time="82.509" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber">
                <failure type="java.lang.AssertionError"
                  message="CDR目的中继检测 【预期：SPS1 实际：FXO7】"
              >
<![CDATA[java.lang.AssertionError: CDR目的中继检测 【预期：SPS1 实际：FXO7】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:269)
  com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber.E4_checkPhyAndVoip(EmergencyNumber.java:659)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="B1_addEmergencyNum" time="31.747" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F2_EventCenter3" time="11.494" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="E2_checkVoipAndPhy" time="52.411" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="A1_registerExtensions" time="51.421" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="E5_PhyAndPhy" time="35.539" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="G7_delteAll_yes" time="13.271" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="G2_deleteOne_no" time="15.174" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="D7_check_gsm" time="0.000" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="D5_check_pstn" time="56.287" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="C6_checkTrunk" time="71.934" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="G3_deleteOne_yes" time="24.726" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="D4_check_spx" time="52.102" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="C4_checkTrunk1" time="52.025" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="C7_deleteNotification" time="26.216" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="D6_check_e1" time="0.000" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F2_EventCenter4" time="8.017" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F2_eventCenter7" time="10.060" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F2_eventCenter8" time="71.066" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="C1_edittrunk_Unavail" time="34.242" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F2_EventCenter2" time="71.729" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F2_EventCenter1" time="10.011" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F1_Contacts2" time="71.517" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="B4_makeCall1" time="11.082" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F1_Contacts3" time="22.123" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="E1_voipAndPhysical" time="44.333" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="B4_makeCall2" time="52.309" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="D3_check_iax" time="52.479" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="B2_makeCall" time="52.037" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="C8_checkNotification" time="53.070" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="C2_checkUnavailTrunk" time="32.654" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="G6_deleteAll_no" time="9.711" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F2_eventCenter6" time="13.264" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="H_recovery" time="43.635" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="D1_addEmergency_line" time="99.654" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="C5_deleteTrunk_addNotification" time="30.678" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="B3_editEmergencyNumb" time="34.426" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="G1_addNumber" time="3.651" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F1_Contacts1" time="27.210" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="A2_addUnavailTrunk" time="21.628" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="E3_PhyAndVoip" time="35.587" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F2_eventCenter9" time="24.228" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="G4_deletePart_no" time="18.346" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="F3_recoveryEvent" time="19.918" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="D5_check_bri" time="52.396" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="C3_addTrunk" time="45.131" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
                <testcase name="G5_deletePart_yes" time="13.338" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.EmergencyNumber" />
      
</testsuite>
